src/components/CodeMirror/index.jsx

```diff
@@ -1,35 +1,11 @@
-import React, { lazy, Suspense } from 'react';
-import PropTypes from 'prop-types';
+import React, { lazy } from 'react';
 
 const Component = lazy(() => import('./CodeMirror'));
 
 export default class CodeMirror extends React.Component {
-  static propTypes = {
-    code: PropTypes.string,
-    mode: PropTypes.string,
-    onChange: PropTypes.func
-  };
-
-  static defaultProps = {
-    code: '',
-    onChange: () => {},
-    mode: 'yaml'
-  };
-
   render() {
-    const {
-      onChange, code, mode, ...rest
-    } = this.props;
+    const { props } = this;
 
-    return (
-      <Suspense fallback={<div>Loading...</div>}>
-        <Component
-          value={code}
-          onChange={onChange}
-          options={{ mode, lineNumbers: true }}
-          {...rest}
-        />
-      </Suspense>
-    );
+    return <Component {...props} />;
```

Comment:
react lazy should work with suspense, are you sure this can work ?

Reply:
ok